hw/ip/otbn/util/rig/program.py

```diff
@@ -346,6 +346,13 @@ def pick_branch_targets(self,
 
         '''
 
+        # Round tgt_min and tgt_max inwards to multiples of 4 (because
+        # instructions always have to be word aligned)
+        if tgt_min is not None:
+            tgt_min = (tgt_min + 3) & ~3
```

Comment:
Shouldn't that be ~2 (or am I confused and should go on a christmas break?)

Reply:
I think it might be the latter :-) ~3 has every bit set except the bottom two...

Reply:
Uh, yes. My brain was thinking "bottom three bits" repeatedly, which is clearly an indication for "needs a break".